tests/TestProcessor.cpp

```diff
@@ -159,61 +159,61 @@ TEST(TestPWProcessor, TestInvalidUTF8Input)
     ddwaf_destroy(handle);
 }
 
-TEST(TestPWProcessor, TestCache)
-{
-    //Initialize a PowerWAF rule
-    auto rule = readFile("processor2.yaml");
-    ASSERT_TRUE(rule.type != DDWAF_OBJ_INVALID);
+//TEST(TestPWProcessor, TestCache)
```

Comment:
Is this commented out intentionally?

Reply:
I mean, the whole file

Reply:
The test is commented out because I need to replace it but I want to keep it as a reference as I'll do that soon on a future PR.